index.js

```diff
@@ -56,6 +56,11 @@ if (argv.watch) {
   process.stdin.resume()
 }
 
+/* istanbul ignore next */
+if (parseInt(postcss().version) < 8) {
+  error('Please install PostCSS 8 or above')
```

Comment:
It would be so nice if we sill suggest the CLI command to fix the problem: `npm i postcss` or `yarn add postcss`

Reply:
So something like maybe? Writing copy isn't my strong suit.
```suggestion
  error(`Please install PostCSS 8 or above.

Install with NPM:
  npm install postcss

Install with yarn:
   yarn add postcss`)
```

Reply:
Also, should it be `install postcss` or be specific about the version with `install postcss@^8`?

Reply:
Too long. The best option is auto-detection by looking for `yarn.lock`. There are 2 options:

Maybe we can use some library to find `package.json` and then check `yarn.lock`. `postcss-cli` already should use some library in `cosmiconfig` for this.

Reply:
> Also, should it be install postcss or be specific about the version with install postcss@^8?

Just latest version will be enouph

Reply:
> Maybe we can use some library to find `package.json` and then check `yarn.lock`.

That could be achieved by using the [`fs.access`](https://nodejs.org/docs/latest-v10.x/api/fs.html#fs_fs_access_path_mode_callback) method right, it's available `node >= 10`.

```js
const file = 'yarn.lock';

fs.access(file, fs.constants.F_OK, (err) => {
  console.log(`${file} ${err ? 'does not exist' : 'exists'}`);
});
```

Then set an internal check if the end-user uses yarn or npm based on if the `yarn.lock` exists and display suggestions based on that.

Reply:
You do not know the current dir to be sure that `postcss` was called from the same dir with `yarn.lock`.

`yarn.lock` can be in the folder above.